Sources/Error.swift

```diff
@@ -2,10 +2,13 @@ public enum Error: ErrorType, Equatable {
     case General
     case InvalidUrl(String)
     case InvalidResponseData(NSData?)
-    case ErrorNS(NSError?)
-    case Error(ErrorType)
     case InvalidAuthentication
     case ShouldOverride
+    case Error(domain: String, code: Int, userInfo: [NSObject: AnyObject])
+}
+
+public func errorFromNSError(nsError: NSError) -> Error {
```

Comment:
🎈To Objective-C'ig. Ik zou meer `error(fromNSError nsError: NSError) -> Error` doen.